<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>askPause</class>
 <widget class="QWidget" name="askPause">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">left: 0px;
top: 0px;
width: 1280px;
height: 720px;
background-color: rgba(255, 255, 255, 200);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>147</y>
     <width>900</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
background-color: rgba(45, 44, 43, 255);
border: 2px solid rgba(45, 44, 43, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>288</y>
     <width>1008</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;font-family:Barlow;border-radius: 20px;
background-color: rgb(255, 99, 11);
color: rgba(255, 255, 255, 255);
font-size: 36px;
text-align: center;
font-family: Barlow;
border: 2px solid rgba(255, 99, 11, 255);</string>
   </property>
   <property name="text">
    <string>STOP PRINT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>152</y>
     <width>1008</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;font-family:Barlow;border-radius: 20px;
background-color: rgb(255, 99, 11);
color: rgba(255, 255, 255, 255);
font-size: 36px;
text-align: center;
font-family: Barlow;
border: 2px solid rgba(255, 99, 11, 255);</string>
   </property>
   <property name="text">
    <string>PAUSE PRINT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>424</y>
     <width>1008</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;font-family:Barlow;border-radius: 20px;
background-color: rgb(255, 99, 11);
color: rgba(255, 255, 255, 255);
font-size: 36px;
text-align: center;
font-family: Barlow;
border: 2px solid rgba(255, 99, 11, 255);</string>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(45, 44, 43, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
